src/content/validated-patterns/index.json

```diff
@@ -0,0 +1,17 @@
+{
+	"sidebarCategories": [],
+	"landingPage": {
+		"hero": {
+			"heading": "Validated Patterns",
+			"image": "/img/well-architected-framework/landing-hero-graphic.png"
```

Comment:
```suggestion
```

Reply:
I think we need image, the build is failing without it:

```
Type error: Type '{ metadata: { title: string; name: string; slug: string; }; data: { pageData: { blocks: ProductViewBlock[]; showProductSitemap?: boolean; }; validatedPatternsContent: { overview: { ...; }; hero: { ...; }; }; }; outlineItems: OutlineLinkItem[]; layoutProps: { ...; }; }' is not assignable to type 'ValidatedPatternsLandingProps'.
  The types of 'data.validatedPatternsContent.hero' are incompatible between these types.
    Property 'image' is missing in type '{ heading: string; }' but required in type 'HeroHeadingVisualProps'.
```

https://vercel.com/hashicorp/dev-portal/EBgJtRzAbM72KKaq8NKmkR7HyhbA?filter=errors#L91